### ISSUE TITLE
Install additional dependencies for given CLI

### ISSUE THREAD

**Issue by vlcinsky** [cs01/pipx#49]
[python-language-server](https://github.com/palantir/python-language-server) allows installing some additional features using extensions, e.g.:

```$ pipx install python-language-server[all]```
includes support for Rope, Pyflakes, McCabe, pycodestyle, pydocstyle, autopep8 and YAPF.

However, some plugins must be installed separately into given virtualenv:

- pyls-mypy Mypy type checking for Python 3
- pyls-isort Isort import sort code formatting
- pyls-black for code formatting using Black

Question: Does pipx allow installing additional python packages into virtualenv of installed tool?

**Comment by cs01** [comment]
Yes and there is going to be an easier way to do it in the near future. Take a look at https://github.com/cs01/pipx/pull/45 and feel free to add any thoughts you have.
